FromZeroToHero.sln

```diff
@@ -1,15 +1,177 @@
 ﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31005.135
-MinimumVisualStudioVersion = 10.0.40219.1
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{97C2C66B-7A7A-4549-BC48-643938A17678}"
-EndProject
-Global
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {7B4CEAD3-9259-48FC-8FE9-D1106B5A8C3E}
-	EndGlobalSection
-EndGlobal
+using System;
+
+namespace Homework2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] numbers = new int[] { 10, 5, 9, 87, 56 };
+            //SortArray(numbers);
+            //AddElementAtStart(numbers, 12);
+            // AddElementAtEnd(numbers,35);
+            // AddElementAtAnyPosition(numbers, 33, 4);
+            //RemoveElementAtStart(numbers);
+            //RemoveElementAtEnd(numbers);
+            // RemoveElementAtPositon(numbers, 3);
+            LoginInfo("KIN", "21lome");
+
+
+        }
+
+        private static void SortArray(int[] numbers)
```

Comment:
Interesting choice on making the methods `private`.  We haven't gone over this concept yet but I would say it is appropriate here.